### DISCUSSION TITLE
csh can't use FZF

### DISCUSSION THREAD

**Discussion by luohongting** [junegunn/fzf#3001] [Q&A] [answered]
how to use FZF in a csh?

**Comment by graahnul-grom** [comment, marked as answer]
I use my own scripts to work with `tcsh` command history (`Ctrl+R`):
https://github.com/graahnul-grom/fzf-csh

**Reply by luohongting** [reply]
thank you very much!

**Reply by graahnul-grom** [reply]
> thank you very much!

You're welcome. Good luck.
